content/docs-v1/multiformat-output.md

```diff
@@ -13,40 +11,27 @@ Quire is designed to create a website version, a PDF version and two e-book vers
 
 ## Site Output
 
-Create the HTML files for your project by running `quire site` in your command-line shell. The files will be built into your project’s `site` folder along with all the necessary static assets like image files, stylesheets and script files. The `site` file will be updated and overwritten each time you run `quire site`.
+Create the HTML files for your project by running `quire build` in your command-line shell. The files will be built into your project’s `_site` folder along with all the necessary static assets like image files, stylesheets and script files. The `_site` file will be updated and overwritten each time you run `quire build`.
 
-{{< q-class "box tip" >}}
-- If you are including PDF and e-book downloads as part of your online site (this is Quire’s default) you’ll need to update those files by running `quire epub` and `quire pdf` as described below **before** running `quire site`.
-{{< /q-class >}}
-
-You can hide specific pages of your project from the site output by adding `online: false` to the page YAML. Or conversely, you can add pages exclusively to the site output, by adding `epub: false` and `pdf: false` to the page YAML and leaving the `online` attribute unset, or set to `true`.
+If you are including PDF and e-book downloads as part of your online site (this is Quire’s default) you’ll need to update those files by running `quire epub` and `quire pdf` as described below **before** running `quire build`.
```

Comment:
Oh, I didn't realize you had to run `quire pdf` and `quire epub` **before** `build`. Is this new? Or did I just misunderstand Matthew before? Does this mean if you were creating both a PDF and E-book the process would be: 

quire pdf
quire build
quire epub
quire build

Reply:
Oh, shoot. Well this is problematic. To build the pdf, you were right, you have to run `quire build` and then `quire pdf` However, if you want to include the PDF as part of the built site (as a download) you have to move the PDF file you just created into your `contents/_assets/downloads` directory and then run `quire build` again. That's ... not great.

1. quire build
2. quire pdf
3. quire epub
4. Move the PDF and EPUB files
5. quire build

Ooof.